release-1.5/release-notes-draft.md

```diff
@@ -5,12 +5,12 @@
 ## Major Themes
 
 - Improved Federation Support
+  - New command: `kubefed`
```

Comment:
@nikhiljindal it is mentioned in the `Features` section below. I see this section more or less like table of contents.